themes/codefor-theme/assets/scss/main.scss

```diff
@@ -333,3 +333,10 @@ blockquote {
 .lab-logo {
   width: 200px;
 }
+
+.landing-callout {
+  border: 2px solid $codefor-green;
+  background-color: lighten($codefor-green, 35%);
```

Comment:
```suggestion
  background-color: lighten($codefor-green, 78%);
```

Reply:
can we make it a bit lighter, to increase the contrast? i tried the 78% [in this tool](https://mdigi.tools/lighten-color/#00da88) and this way the lightened color is #BFFFE7 which looks almost identical to $codefor-mint: #BFEEEB;

Reply:
Nice, this looks much more consistent with the mint color! ✨